lib/journaled.rb

```diff
@@ -1,14 +1,13 @@
 require "aws-sdk-resources"
-require "delayed_job"
 require "json-schema"
 require "request_store"
 
 require "journaled/engine"
-require "journaled/job_priority"
+require 'journaled/enqueue'
 
 module Journaled
   mattr_accessor :default_app_name
-  mattr_accessor(:job_priority) { Journaled::JobPriority::EVENTUAL }
+  mattr_accessor(:job_priority) { 20 }
```

Comment:
This might be the most controversial change. Since the README makes no mention of job priority constants, and since I was attempting to de-emphasize the dependency on DJ (and push the `job_priority` option down into a very specific default implementation of `enqueue`), I felt like having named priorities was superfluous to the function of this gem. (I could see this `job_priority` default eventually getting tucked into a `Journaled::DelayedJob` namespace.)

But I'm happy to discuss.